src/v/archival/service.cc

```diff
@@ -426,6 +427,17 @@ s3::bucket_name scheduler_service_impl::get_bucket() const {
     return _conf.bucket_name;
 }
 
+uint64_t scheduler_service_impl::estimate_backlog_size() {
+    ss::gate::holder g(_gate);
```

Comment:
gate is not needed, this part is synchronous.

Reply:
removed it